Content.Shared/Xenoarchaeology/Artifact/XAT/Components/XATTimerComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Xenoarchaeology.Artifact.XAT.Components;
+
+/// <summary>
+/// This is used for a xenoarch trigger that self-activates at a regular interval
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(XATTimerSystem)), AutoGenerateComponentState, AutoGenerateComponentPause]
+public sealed partial class XATTimerComponent : Component
+{
+    [DataField, AutoNetworkedField, AutoPausedField]
+    public TimeSpan NextActivation;
```

Comment:
Only a regular interval and not a random one?

Reply:
don't know why Emo left it this way. Maybe better to let it be min-max too?... gonna look into this...

Reply:
fixed that thingy, a bit at least...